src/ServiceControl/Recoverability/Grouping/DomainHandlers/StoreHistoryHandler.cs

```diff
@@ -35,7 +35,7 @@ public void Handle(RetryOperationCompleted message)
                 retryHistory.AddToUnacknowledged(new UnacknowledgedRetryOperation
                 {
                     RequestId = message.RequestId,
-                    RetryType = message.RetryType,
+                    OperationType = message.RetryType,
```

Comment:
To distinguish the retry type. Group, single message, multiple messages, etc. I don't think it's actually used anywhere though.